Source/SVWidgetsLib/Widgets/ImportASCIIDataWizard/ImportASCIIDataWizard.cpp

```diff
@@ -275,11 +276,11 @@ QStringList ImportASCIIDataWizard::ReadLines(const QString& inputFilePath, int b
   if(inputFile.open(QIODevice::ReadOnly))
   {
     QTextStream in(&inputFile);
-
+    QString aLine;
     for(int i = 1; i < beginLine; i++)
     {
       // Skip all lines before "value"
-      in.readLine();
+      aLine = in.readLine();
     }
```

Comment:
I understand why aLine was created, but it took some digging to figure out the assumption that the last value of aLine matters because we bother to waste memory on it was invalid.  Just use `(void)in.readLine();`  We already use `(void)x;` several places in the code as it is the definition of Q_UNUSED, and it's faster to read than trying to figure out why I care about aLine.

Reply:
I think that was left over from debugging?